nvme.c

```diff
@@ -2413,6 +2413,60 @@ static int virtual_mgmt(int argc, char **argv, struct command *cmd, struct plugi
 	return nvme_status_to_errno(err, false);
 }
 
+static int primary_ctrl_caps(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+{
+	const char *desc = "Send an Identify Primary Controller Capabilities "\
+		"command to the given device and report the information in a "\
+		"human-redable or binary format.";
+	const char *raw = "show capabilities in binary format";
+	const char *human_readable = "show capabilities in readable format";
+
+	enum nvme_print_flags flags = NORMAL;
+	struct nvme_primary_ctrl_caps caps;
+	int err, fd;
+
+	struct config {
+		int raw_binary;
+		int human_readable;
+		char *output_format;
+	};
+
+	struct config cfg = {
+		.output_format = "normal",
+	};
+
+	OPT_ARGS(opts) = {
+		OPT_FMT("output-format",   'o', &cfg.output_format,  output_format),
+		OPT_FLAG("raw-binary",     'b', &cfg.raw_binary,     raw),
```

Comment:
This is redundant with the output-format option. Older commands have the option only for backward compatibility.